pymatgen/io/phonopy.py

```diff
@@ -430,3 +450,246 @@ def get_phonon_band_structure_symm_line_from_fc(
     labels_dict = {a: k for a, k in zip(labels, kpoints) if a != ""}
 
     return PhononBandStructureSymmLine(kpoints, frequencies, structure.lattice, labels_dict=labels_dict)
+
+
+def get_gruneisenparameter(gruneisen_path, structure=None, structure_path=None) -> GruneisenParameter:
+    """
+    Get Gruneisen object from gruneisen.yaml file, as obtained from phonopy (Frequencies in THz!).
+    The order is structure > structure path > structure from gruneisen dict.
+    Newer versions of phonopy include the structure in the yaml file,
+    the structure/structure_path is kept for compatibility.
+
+    Args:
+        gruneisen_path: Path to gruneisen.yaml file (frequencies have to be in THz!)
+        structure: pymatgen Structure object
+        structure_path: path to structure in a file (e.g., POSCAR)
+
+    Returns:
```

Comment:
This docstring also needs the "Returns:" section specified.

Reply:
Added returns GruneisenParameter object